docs/project/find-an-issue.md

```diff
@@ -122,14 +118,14 @@ In this section, you find and claim an open documentation lines issue.
 
     We don't allow external contributors to assign issues to themselves. So, you
```

Comment:
I'm trying to kill all the "We's" so maybe  "The project doesn't permit external..."  If you are willing to make that change.